backend/src/routes/api/namespaces/namespaceUtils.ts

```diff
@@ -16,6 +18,36 @@ export const applyNamespaceChange = (
       400,
     );
   }
+  const kc = fastify.kube.config;
+  const cluster = kc.getCurrentCluster();
+
+  const selfSubjectAccessReviewResult = await passThrough<V1SelfSubjectAccessReview>(
+    fastify,
+    request,
+    {
+      url: `${cluster.server}/apis/authorization.k8s.io/v1/selfsubjectaccessreviews`,
+      method: 'POST',
+      requestData: JSON.stringify({
+        apiVersion: 'authorization.k8s.io/v1',
+        kind: 'SelfSubjectAccessReview',
+        spec: {
+          resourceAttributes: {
+            group: 'project.openshift.io',
+            resource: 'projects',
+            subresource: '',
+            verb: 'update',
+            name,
+            namespace: name,
```

Comment:
Do you need to pass namespace 🤔

Reply:
Yes, I think so. I tried to remove the namespace and the functionality doesn't work any more, check the comment here https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/rbac.tsx#L48-L49

Reply:
Good catch! Thanks